tiledb/sm/stats/global_stats.cc

```diff
@@ -0,0 +1,878 @@
+/**
+ * @file   global_stats.cc
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2018-2021 TileDB, Inc.
```

Comment:
2021

Reply:
That's the old `stats.cc` file, so let's keep the copyright as is.